### ISSUE TITLE
EquipmentSets index should start at 0

### ISSUE THREAD

**Issue by lynxhug** [tg123/myslot#10]
I got some problems with saving/restoring my equipment sets with this addon. In process of pinpointing issue after some deleting and recreating new sets, problem with saving/restoring went away for some reason. Nonetheless, it seems as equipment sets are indexed starting from 0 to `GetNumEquipmentSets() - 1`. For example:
`/run for i = 0, C_EquipmentSet.GetNumEquipmentSets()-1 do n, icon, index=C_EquipmentSet.GetEquipmentSetInfo(i);print(n,icon,index," -- ",i) end`
Gives correct output:
```
Feral 132402 0  --  0
Heal 132378 1  --  1
test 254105 2  --  2
```
`/run for i = 1, C_EquipmentSet.GetNumEquipmentSets() do n, icon, index=C_EquipmentSet.GetEquipmentSetInfo(i);print(n,icon,index," -- ",i) end`
Does not:
```
Heal 132378 0  --  1
test 254105 1  --  2
nil nil nil  --  3
```
Not sure if i referenced right line to correct, as it seems I am unable to recreate issue anymore.
https://github.com/tg123/myslot/blob/c1c40eb176fca97a3cf3fba163058cbc6680e118/Myslot.lua#L121

**Comment by tg123** [comment]
i think it is ok to enumerate from 0
myslot will check if get(i) == expected

**Comment by tg123** [comment]
fixed 5.4.2
